### PR TITLE
feat: add support for sending events for heroku/api

### PR DESCRIPTION
**Pull request by abtris**
Need add tests

### CODE REVIEW

**Comment by abtris** [comment]
@w-vi I'm not sure about testing.

Event have format: `_e{4,46}:test|Scale to mailer=1, web=3 by someuser@gmail.com` but tests are failing.

Can you look what is wrong?

**Comment by w-vi** [comment]
You are still sending the metrics not just events https://github.com/apiaryio/heroku-datadog-drain-golang/blob/abtris/datadog-events/client.go#L165 
But the test is not expecting those to receive.

Easy fix is to have the test case like this:

```go
	{
		cnt: 3,
		Req: `222 <134>1 2015-04-07T16:01:43.517062+00:00 host heroku api - Scale to mailer=1, web=3 by someuser@gmail.com`,
		Expected: []string{
			"_e{4,46}:test|Scale to mailer=1, web=3 by someuser@gmail.com",
			"heroku.dyno.mailer:1.000000|g",
			"heroku.dyno.web:3.000000|g",
		},
	},
```

**Comment by abtris** [comment]
@w-vi thanks

File: client.go [hunk 1]
```diff
@@ -159,6 +158,10 @@ func (c *Client) sendScalingMsg(data *logMetrics) {
 		"prefix": *data.prefix,
 	}).Debug("sendScalingMsg")
 
+	for _, v := range data.events {
+		c.SimpleEvent(*data.app, v)
```
Comment (abtris) [inline client.go]: @w-vi I'm not sure about this is `title` can be just app name or more

File: client.go [hunk 2]
```diff
@@ -159,6 +158,10 @@ func (c *Client) sendScalingMsg(data *logMetrics) {
 		"prefix": *data.prefix,
 	}).Debug("sendScalingMsg")
 
+	for _, v := range data.events {
+		c.SimpleEvent("Heroku app: "+*data.app, v)
```
Comment (w-vi) [inline client.go]: Is it necessary to add the "Heroku app: " ? Is not the message itself enough.

Comment (w-vi) [reply client.go]: Maybe add some way how to identify the app in question like  <APP_NAME> from `Oct 09 18:26:54 <APP_NAME> heroku/api:  Scale to web=6 by user@company.com`

Comment (abtris) [reply client.go]: APPNAME we have in `*data.app` and that why I can add into event. And heroku is good to know, you can have events from many systems.

Comment (w-vi) [reply client.go]: In that case maybe better to put `heroku/api` as maybe in the future there will be some event from `heroku/router` or something like that so you know from which part of heroku it comes? what do you think?

Comment (abtris) [reply client.go]: ok, we can change anytime.

Comment (abtris) [reply client.go]: Fixed and this can be ok for first iteration, we can improve after we have some data.

**Pull request merged**